src/modules/iotjs_module_buffer.c

```diff
@@ -174,6 +174,22 @@ size_t iotjs_bufferwrap_copy(iotjs_bufferwrap_t* bufferwrap, const char* src,
   return iotjs_bufferwrap_copy_internal(bufferwrap, src, 0, len, 0);
 }
 
+static size_t bufferslice_idx_setter(int64_t point, size_t len) {
```

Comment:
Perhaps a better name would be `buffer_index_normalizer` and the parameters could be named `index` and `max_length`